lib/private/remove-non-userland-calls-from-stack.js

```diff
@@ -45,6 +45,9 @@ module.exports = function removeNonUserlandCallsFromStack(stackEntries) {
 
   try {
 
+    var libPrivateDir = __dirname;
+    var libDir = path.resolve(libPrivateDir, '..');
```

Comment:
optimization: could pull `libPrivateDir` and `libDir` to the top of this file so this only runs once